docker/fsroot/usr/local/bin/entry

```diff
@@ -72,7 +72,16 @@ source /fwpy/bin/activate
 
 if tty -s; then
     if [ "$#" -eq 0 ]; then
-        tmux
+		while true;
+		do
+			tmux
+			read -p "Exit? (yes/no); " response
```

Comment:
Maybe something a bit more clear? "Exit and shutdown container? (yes/no); " or "Exit and kill container (yes/no); " or "Are you sure you want to end your experiment because the container is dying and you can't get it back (yes/no)"